packages/x-tailwindcss/src/x-tailwind-plugin/components/layout/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the component `layout` utils CSS.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function utils(helpers: TailwindHelpers) {
+  return {
+    '.layout-expand': {
+      height: '100%',
```

Comment:
I'm not sure if this height: '100%' is needed. Please, ping me if I'm wrong & let's check it 🙏